roles/storage-demo/README.md

```diff
@@ -0,0 +1,21 @@
+# Ansible role: storage-demo
+
+**Note: This role is not for production use.  All storage is erased any time**
```

Comment:
s/any time/every time

Reply:
I think these are interchangeable in this context so can we keep it if this is the only requested change?  I can reword if I respin the patch for another reason.